udemy_exercises/bank/Main.java

```diff
@@ -0,0 +1,23 @@
+package udemy_exercises.bank;
+
+public class Main {
+    public static void main(String[] args) {
+        Bank bank = new Bank("J.P. Morgan Chase");
+
+        bank.addBranch("Frisco");
+        bank.addCustomer("Frisco", "John Doe", 500.59);
+        bank.addCustomer("Frisco", "Ben Dover", 420.69);
+
+        bank.addBranch("Prosper");
+        bank.addCustomer("Prosper", "Sofa King", 420.00);
```

Comment:
Remember this repo is something potential employers will look at as long as it's up...funny, but keep it professional